src/tapis-app/Workflows/_components/Toolbar/AddGroupUsersModal/AddGroupUsersModal.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useCallback } from 'react';
+import { Button } from 'reactstrap';
+import { SubmitWrapper } from 'tapis-ui/_wrappers';
+import { Form, Formik, FieldArray, Field } from 'formik';
+import { FormikInput, GenericModal, Icon } from 'tapis-ui/_common';
+import { focusManager } from 'react-query';
+import { useCreate } from 'tapis-hooks/workflows/groupusers';
+import styles from './AddGroupUsersModal.module.scss';
+import { Workflows } from '@tapis/tapis-typescript';
+import * as Yup from 'yup';
+
+type AddGroupUserModalProps = {
+  toggle: () => void;
+  groupId: string;
+};
+
+const AddGroupUsersModal: React.FC<AddGroupUserModalProps> = ({
+  toggle,
+  groupId,
+}) => {
+  const { create, isLoading, error, isSuccess } = useCreate();
+  const onSuccess = useCallback(() => {
+    // Calling the focus manager triggers react-query's
+    // automatic refetch on window focus
+    focusManager.setFocused(true);
```

Comment:
The idiomatic way to do what (I think) you're trying to do here is to use query invalidation to cause a refresh of the specific query you want: https://react-query-v3.tanstack.com/guides/query-invalidation

Reply:
I just tried it and it works. I will replace all uses of focusManager with query invalidation

Reply:
Turns out, it is not the solution for every instance in which we use focus manage, but it seems to work for all cases in workflows

Reply:
Replaced instances of focusManager set focus with query invalidation